champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/warlock/axe/Clone.java

```diff
@@ -253,6 +256,26 @@ public void onCustomDamageEvent(CustomDamageEvent event) {
         }
     }
 
+    @EventHandler
+    public void onEntityPropertyEvent(GetEntityRelationshipEvent event) {
+        if (!(event.getTarget() instanceof Vindicator vindicator)) return;
+
+        final Player owner = getCloneOwner(vindicator);
+        if (!clones.containsKey(owner)) return;
+        event.setEntityProperty(UtilEntity.getRelation(event.getEntity(), owner));
```

Comment:
UtilEntity#getRelation calls GetEntityRelationship event, so this will be a stack overflow